src/contexts/wishlistContext.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import { useState, useEffect, createContext, useContext } from "react";
+import { useAuth } from "./authContext";
+
+const WishlistContext = createContext();
+
+const useWishlist = () => useContext(WishlistContext);
+
+const WishlistProvider = ({ children }) => {
+  const [wishlist, setWishlist] = useState([]);
+  const {
+    auth: { isAuthenticated, token },
+  } = useAuth();
+
+  useEffect(() => {
+    if (isAuthenticated) {
+      (async () => {
+        try {
+          const response = await axios.get("/api/user/wishlist", {
```

Comment:
I just found out if you log out the items will still be on the wishlist.
![bug](https://user-images.githubusercontent.com/85923879/159674891-5d0f1c31-f612-4198-89a3-1a7dbbf0fae7.png)

So instead of separate get call here what you can do is use data which you get in response from `api/auth/login`

Reply:
Thanks for pointing it out, fixed now.